lib/util/mp4_box_parsers.js

```diff
@@ -498,6 +498,24 @@ shaka.util.Mp4BoxParsers = class {
     return {codec};
   }
 
+  /**
+   * Parses a TENC box.
+   * @param {!shaka.util.DataViewReader} reader
+   * @return {!shaka.util.ParsedTENCBox}
+   */
+  static parseTENC(reader) {
+    reader.readUint8(); // reserved
+    reader.readUint8();
+    reader.readUint8(); // default_isProtected
+    reader.readUint8(); // default_Per_Sample_IV_Size
+    let defaultKID = '';
+    for (let i = 0; i <16; i++) {
+      const hex = reader.readUint8().toString(16);
+      defaultKID += (hex.length === 1 ? '0' + hex : hex);
+    }
```

Comment:
How about you `readBytes(16)` into a Uint8Array and the run toHex?

Reply:
Changed!